public/views/SelectTarget.vue

```diff
@@ -340,22 +340,13 @@ export default Vue.extend({
 .available-target-variables /deep/ .variable-facets {
   height: 100%;
 }
-
-/* Render items as columns */
 .available-target-variables /deep/ .variable-facets-container {
   column-count: 3;
   column-gap: 1rem;
 }
-
 .available-target-variables /deep/ .variable-facets-item {
```

Comment:
Yes. By leaning more on the basic bootstrap layout classes, we don't have to take care of that styling any more. It all just works because it's own rules nested in it's own rules.